### PR TITLE
aic8800 support

### PR DESCRIPTION
**Pull request by 0x754C**

### CODE REVIEW

**Comment by nekorouter** [comment]
Have you add any changes about device tree? I can't find it anywhere.

File: drivers/net/wireless/aic8800/aic8800_fdrv/aicwf_sdio.h [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+typedef enum {
+	SDIO_TYPE_DATA         = 0X00,
+	SDIO_TYPE_CFG          = 0X10,
+	SDIO_TYPE_CFG_CMD_RSP  = 0X11,
+	SDIO_TYPE_CFG_DATA_CFM = 0X12,
+	SDIO_TYPE_CFG_PRINT    = 0X13
+} sdio_type;
+
+/* SDIO Device ID */
+#define SDIO_VENDOR_ID_AIC8801                0x5449
```
Comment (0x754C) [inline drivers/net/wireless/aic8800/aic8800_fdrv/aicwf_sdio.h:70]: SDIO VENDOR ID

**Comment by 0x754C** [comment]
sdio wifi not need device tree:

```
https://github.com/revyos/thead-kernel/pull/36/commits/4d48cc5362998daf7ea1105273bc3e6640cbece4#r1333765443
```

**Comment by 0x754C** [comment]
close pr, vendor(aic) provide new version driver. we need merge it.

**Pull request closed without merging**